languages/fsharp/exercises/concept/arrays/.docs/instructions.md

```diff
@@ -0,0 +1,58 @@
+You're an avid bird watcher that keeps track of how many birds have visited your garden in the last seven days.
+
+You have six tasks, all dealing with the numbers of birds that visited your garden.
+
+### 1. Check what the counts were last week
+
+For comparison purposes, you always keep a copy of last week's counts nearby, which were: 0; 2; 5; 3; 7; 8 and 4. Define the `lastWeek` binding that contains last week's counts:
```

Comment:
Since this is an English sentence and not code, commas seem more natural

```suggestion
For comparison purposes, you always keep a copy of last week's counts nearby, which were: 0, 2, 5, 3, 7, 8, and 4. Define the `lastWeek` binding that contains last week's counts:
```